ckanext/canada/helpers.py

```diff
@@ -175,6 +175,21 @@ def portal_url():
 
 def googleanalytics_id():
     return str(config.get('googleanalytics.id'))
+
+def adobe_analytics_login_required():
+    return "2" #return 1 if page requires a login and 2 if page is public
+
+def adobe_analytics_lang():
+    if h.lang() == 'en':
+        return 'eng'
+    elif h.lang() == 'fr':
+        return 'fra'
+
+def adobe_analytics_js():
+    if config.get('ckan.site_url', '') == 'https://staging.open.canada.ca':
```

Comment:
please use a new configuration option instead of hard-coding the asset locations based on the site_url setting